dll/apisets/api-ms-win-crt-private-l1-1-0.spec

```diff
@@ -633,19 +655,21 @@
 @ stub _o__read
 @ stub _o__realloc_base
 @ stub _o__recalloc
-@ stub -version=0xA00+ _o__register_onexit_function
+@ stub _o__register_onexit_function
 @ stub _o__resetstkoflw
 @ stub _o__rmdir
 @ stub _o__rmtmp
 @ stub _o__scalb
 @ stub _o__scalbf
 @ stub _o__searchenv
 @ stub _o__searchenv_s
-@ cdecl -version=0xA00+ _o__seh_filter_dll() msvcrt.__CppXcptFilter
-@ cdecl -version=0xA00+ _o__seh_filter_exe() msvcrt._XcptFilter
+@ stub _o__seh_filter_dll
+@ stub _o__seh_filter_exe
```

Comment:
Why revert to stub?

Reply:
sorry I skipped the merge of that line, will fix it.